packages/manager/components/settings/createEnv.tsx

```diff
@@ -117,7 +117,7 @@ const CreateEnv: FunctionComponent<ApiKeyProps> = ({ webprop }: AnyProps) => {
       }
       else {
         setAlert([
-          { title: `${propertyRes.response}`, variant: 'danger', key: getUniqueId() },
+          { title: `Unable to process the request, Please try again if this problem persists, please contact admin spaship_support@redhat.com`, variant: 'danger', key: getUniqueId() },
```

Comment:
Can you put this string inside a TSX/js  file as a constant and then use it, instead of declaring the same string  in multiple places, it will increase the maintainability of the application